### ISSUE TITLE
[feature request] show old event pages

### ISSUE THREAD

**Issue by hiroMTB** [LacunaLab/LacunaLab.github.io#52]
Lacuna had a many nice events. But only 3 latest events are visible. It might be nice to have old events page listing up all events in timeline.

**Comment by mrzl** [comment]
i think @genekogan developed something more listy-ish that could do this. when we did the website, back in '15 we decided to only have a few events prominently there.. 
a list of past events could be nice, though!

**Comment by thomesh** [comment]
I think what we really need is a dedicated events page with past, present, and future events.

**Comment by hiroMTB** [comment]
@thomesh 
Do you mean 3 different pages?

@mrzl 
nice idea, we can reuse his list up style. 
Just put a link like "more events" and jump to event page where listing up all events from past to future.

**Comment by thomesh** [comment]
No. I am talking about one events page that shows all these. Similar to the one of Spektrum for example: http://spektrumberlin.de/program/events.html

**Comment by hiroMTB** [comment]
nice, I imagine like that as well.
